src/status_im/contexts/shell/view.cljs

```diff
@@ -23,7 +23,7 @@
     [rn/view
      {:accessible          true
       :accessibility-label :view-id-tracker
-      :style               {:position :absolute :z-index -1}}
+      :style               {:width 1 :height 1 :margin-top -1}}
```

Comment:
As mentioned in the comment above the function, the view-id-tracker is a hidden view used only for e2e and PR for testing by QA.

:margin-top -1, to make sure added height don't affect home screen layout.